<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StationStorageSetDialog</class>
 <widget class="QDialog" name="StationStorageSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(150, 157, 167);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>股道</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(240, 240, 240);
border-radius:4px;
padding:2px 4px;</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>250</width>
             <height>19</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="styleSheet">
            <string notr="true">QPushButton{background-color: rgb(240, 240, 240);}
QPushButton{border:1px groove gray;border-radius:4px;padding:2px 4px;border-style:outset;}
QPushButton:hover{background-color: rgb(229, 241, 251);color:black}
QPushButton:pressed{background-color: rgb(204, 228, 247);border-style:inset}
QPushButton{background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(250, 250, 250, 255));}</string>
           </property>
           <property name="text">
            <string>清除</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="styleSheet">
            <string notr="true">QPushButton{background-color: rgb(240, 240, 240);}
QPushButton{border:1px groove gray;border-radius:4px;padding:2px 4px;border-style:outset;}
QPushButton:hover{background-color: rgb(229, 241, 251);color:black}
QPushButton:pressed{background-color: rgb(204, 228, 247);border-style:inset}
QPushButton{background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(250, 250, 250, 255));}</string>
           </property>
           <property name="text">
            <string>确认设置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="styleSheet">
            <string notr="true">QPushButton{background-color: rgb(240, 240, 240);}
QPushButton{border:1px groove gray;border-radius:4px;padding:2px 4px;border-style:outset;}
QPushButton:hover{background-color: rgb(229, 241, 251);color:black}
QPushButton:pressed{background-color: rgb(204, 228, 247);border-style:inset}
QPushButton{background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(250, 250, 250, 255));}</string>
           </property>
           <property name="text">
            <string>关闭</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>股道存车占用信息</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="textBrowser">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:12pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:9pt; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(222, 226, 229, 250);</string>
        </property>
        <property name="title">
         <string>存车信息</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5"/>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="styleSheet">
         <string notr="true">




</string>
        </property>
        <property name="title">
         <string>编辑站存车</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>24</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>类别</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>20</y>
           <width>60</width>
           <height>20</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(240, 240, 240);
border-radius:4px;
padding:2px 4px;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>20</y>
           <width>24</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>车种</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_2">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>20</y>
           <width>80</width>
           <height>20</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(240, 240, 240);
border-radius:4px;
padding:2px 4px;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>20</y>
           <width>48</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>作业说明</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_3">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>20</y>
           <width>60</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(240, 240, 240);
border-radius:4px;
padding:2px 4px;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>24</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>辆数</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>60</y>
           <width>48</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>运用标记</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>54</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>说明/去向</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_5">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>60</y>
           <width>151</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(240, 240, 240);
border-radius:4px;
padding:2px 4px;</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>61</y>
           <width>60</width>
           <height>18</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(240, 240, 240);
border-radius:4px;
padding:2px 4px;</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>100</y>
           <width>241</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(240, 240, 240);
border-radius:4px;
padding:2px 4px;</string>
         </property>
        </widget>
        <widget class="QPushButton" name="unlockButton_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>140</y>
           <width>70</width>
           <height>20</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{background-color: rgb(240, 240, 240);}
QPushButton{border:1px groove gray;border-radius:4px;padding:2px 4px;border-style:outset;}
QPushButton:hover{background-color: rgb(229, 241, 251);color:black}
QPushButton:pressed{background-color: rgb(204, 228, 247);border-style:inset}
QPushButton{background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(250, 250, 250, 255));}
</string>
         </property>
         <property name="text">
          <string>插入</string>
         </property>
        </widget>
        <widget class="QPushButton" name="unlockButton_4">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>140</y>
           <width>70</width>
           <height>20</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{background-color: rgb(240, 240, 240);}
QPushButton{border:1px groove gray;border-radius:4px;padding:2px 4px;border-style:outset;}
QPushButton:hover{background-color: rgb(229, 241, 251);color:black}
QPushButton:pressed{background-color: rgb(204, 228, 247);border-style:inset}
QPushButton{background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(250, 250, 250, 255));}
</string>
         </property>
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
        <widget class="QPushButton" name="unlockButton_6">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>140</y>
           <width>70</width>
           <height>20</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{background-color: rgb(240, 240, 240);}
QPushButton{border:1px groove gray;border-radius:4px;padding:2px 4px;border-style:outset;}
QPushButton:hover{background-color: rgb(229, 241, 251);color:black}
QPushButton:pressed{background-color: rgb(204, 228, 247);border-style:inset}
QPushButton{background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(250, 250, 250, 255));}
</string>
         </property>
         <property name="text">
          <string>增加</string>
         </property>
        </widget>
        <widget class="QPushButton" name="unlockButton_5">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>140</y>
           <width>70</width>
           <height>20</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{background-color: rgb(240, 240, 240);}
QPushButton{border:1px groove gray;border-radius:4px;padding:2px 4px;border-style:outset;}
QPushButton:hover{background-color: rgb(229, 241, 251);color:black}
QPushButton:pressed{background-color: rgb(204, 228, 247);border-style:inset}
QPushButton{background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(250, 250, 250, 255));}
</string>
         </property>
         <property name="text">
          <string>修改</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
